libs/fluid-elements/tab-group/src/lib/tab-group.ts

```diff
@@ -107,38 +122,58 @@ export class FluidTabGroup extends LitElement {
         index = this.tabChildren.length - 1;
       }
 
+      // Adds a class that activates the outline when focused
+      this.tabChildren[index].tabbed = true;
       this.tabChildren[index].focus();
-      this.tabChildren[index].tabindex = 0;
-      this.tabChildren[oldIndex].tabindex = -1;
-      this.activetabid = this.tabChildren[index].tabid;
+      this._toBeActiveTab = this.tabChildren[index].tabid;
+      // Set the old to normal state
+      this.tabChildren[oldIndex].tabbed = false;
+    }
+  }
+
+  /** In order to prevent the browser to scroll when the user selects a tab using the spacebar we prevent the default behavior. */
+  private handleKeyDown(event: KeyboardEvent): void {
+    if (event.code === SPACE) {
+      event.preventDefault();
     }
```

Comment:
This is not really a JSDoc annotation about the function, but a code comment that is helpful to the developer of the component to understand why this was necessary. 

JSDoc will often show up for consumers using an IDE (not for private fields obviously), and for them this information would not be very helpful.
```suggestion
  private handleKeyDown(event: KeyboardEvent): void {
     // In order to prevent the browser to scroll when the user selects a tab using the spacebar
     // we prevent the default behavior.
    if (event.code === SPACE) {
      event.preventDefault();
    }
```